### ISSUE TITLE
exa segfaults when symlink is broken

### ISSUE THREAD

**Issue by mrusme** [ogham/exa#983]
If exa does something unexpected, or its output looks wrong, or it displays an error on the screen, or if it outright crashes, then please include the following information in your report:

- The version of exa being used (`exa --version`)
```
*  sys-apps/exa
      Latest version available: 0.9.0-r1
      Latest version installed: 0.9.0-r1
      Size of files: 16,549 KiB
      Homepage:      https://the.exa.website/
      Description:   A modern replacement for 'ls' written in Rust
      License:       Apache-2.0 MIT Unlicense
```

- The command-line arguments you are using
see below

- Your operating system and hardware platform
Gentoo Linux

If it’s a crash, please include the full text of the crash that gets printed to the screen. If you’re seeing unexpected behaviour, a screenshot of the issue will help a lot.

---
```
thread '<unnamed>' panicked at 'called `Result::unwrap()` on an `Err` value: Os { code: 2, kind: NotFound, message: "No such file or directory" }', src/fs/feature/git.rs:271:22
note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace
[1]    15691 IOT instruction  exa --time-style=long-iso --git --icons -lg -la
```

The folder in which I'm performing an `ls` contains a symlink that points to a file that does not exist anymore.

**Comment by daviessm** [comment]
Can you try version 0.10.1 from the ~ tree? I'm pretty sure the issue is fixed there.

**Comment by mrusme** [comment]
So far 0.9 is the [latest stable](https://packages.gentoo.org/packages/sys-apps/exa) on Gentoo. I assume that one reason for that is that they're looking for a new maintainer for it. :-/

**Comment by daviessm** [comment]
Yes, but there's no problem with using a few "unstable" packages if they solve issues.

**Comment by daviessm** [comment]
On Gentoo, `exa-0.10.1` was stabilised for amd64 yesterday; can you test it and close this bug if the problem's gone?

**Comment by ariasuni** [comment]
Yeah it’s very likely to be #526, fixed by #584.

**Comment by mrusme** [comment]
Upgraded & tested, it seems like it's fixed! Thanks!
